### ISSUE TITLE
Selection methods: a way to unify them?

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange3#6155]
Something to think about/be aware of. Currently, there are three selection methods in play in Orange:
1. *table-like*: Ctrl/Cmd and Shift work normally. Can be found in Data Table.
2. *add-remove*: add and remove options, with explicit command on the right hand side. Upon click add an unselected instance and remove a selected instance. Shift/Command don't do anything. Can be found in Linear Projection.
3. *silent add-remove*: add and remove options as above, but without explicit commands on the right. Shift/Command don't do anything. Can be found in Performance Score (and generally in Evaluate widgets).

Is there a way to unify this or are we happy with the current options?

On a similar note, Test and Score provides a selection in the widget, which doesn't seem to do anything (it doesn't affect the output).

**Comment by janezd** [comment]
Linear projection is a case by itself. Its behavior is different because the list view allows for reordering axes and so forth.

As for point 3, this is mostly limited to cases like selection of models in Evaluation, where the usual action would probably be to show and hide specific models (out of 5 or so - the list will never be long), so the behavior, which is chosen explicitly, is probably convenient for the context.

In other words, thanks for noticing and we'll be careful to generally stick to the first option, and use the third only where appropriate.
